xdp.cpp.user.props

```diff
@@ -8,7 +8,7 @@
   </ItemDefinitionGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)'=='Release'">
     <ClCompile>
-      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
+      <RuntimeLibrary>MultiThreadedDll</RuntimeLibrary>
```

Comment:
There's a better way, and doesn't require vc redist.

Reply:
What is it? 😄

Reply:
Oh, you want the answer? 😃

Reply:
`/NODEFAULTLIB:libucrt.lib /DEFAULTLIB:ucrt.lib`

Reply:
`<Link><AdditionalOptions>/NODEFAULTLIB:libucrt.lib</AdditionalOptions></Link>` might be all you need actually

Reply:
Interesting, let me give that a try.

Reply:
Yeah, this seems like the best of both worlds, especially for debug builds, since we can now share `errno` across binary boundaries without needing to redistribute debug runtime binaries, which isn't permitted by the VC license.